.gitignore

```diff
@@ -22,3 +22,6 @@ boards.local.txt
 *.gcda
 *.o
 *.a
+
```

Comment:
Could we remove this file from the commit, please?

Reply:
I took the liberty of deleting them manually and will merge once the build is done.